assets/js/modules/analytics/components/common/ProfileSelect.test.js

```diff
@@ -37,11 +37,20 @@ apiFetchMock.mockImplementation( ( ...args ) => {
 } );
 
 const setupRegistry = ( { dispatch } ) => {
-	const { id, webPropertyId, accountId } = fixtures.propertiesProfiles.profiles[ 0 ]; // eslint-disable-line sitekit/camelcase-acronyms
-	dispatch( STORE_NAME ).setAccountID( accountId ); // eslint-disable-line sitekit/camelcase-acronyms
-	dispatch( STORE_NAME ).setPropertyID( webPropertyId ); // eslint-disable-line sitekit/camelcase-acronyms
+	const {
+		id,
+		webPropertyId: propertyID, // eslint-disable-line sitekit/camelcase-acronyms
+		accountId: accountID, // eslint-disable-line sitekit/camelcase-acronyms
+	} = fixtures.propertiesProfiles.profiles[ 0 ];
+
+	dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+	dispatch( STORE_NAME ).finishResolution( 'getProperties', [ accountID ] );
+	dispatch( STORE_NAME ).finishResolution( 'getProperties', [ '0' ] );
```

Comment:
Why is that here? Seems like there's something else we need to fix instead... Also, I think we should call `receiveGetAccounts` and `receiveGetProperties` here with applicable fixture values too. While they're technically not used in the component, not passing that data while at the same time marking the resolver resolution as finished causes a state that shouldn't exist.

Reply:
It is required for the `should be disabled when in the absence of an valid account or property ID.` test. I have updated that test to finish this resolution there and added fixture values too.